modules/xcode/xcode_common.lua

```diff
@@ -47,7 +47,14 @@
 		if node.isResource then
 			return "Resources"
 		end
-		return categories[path.getextension(node.name)]
+		local category = categories[path.getextension(node.name)]
+		if (not category)
+			and node.cfg
+			and (node.cfg.kind == p.SHAREDLIB or node.cfg.kind == p.STATICLIB)
+		then
+			category = "Frameworks"
```

Comment:
Perhaps this should be treated like an early return like `isResource`? Is there a situation where a static or shared library wouldn't be in the `Frameworks` category?
```lua
if node.isResource then
  return "Resources"
elseif node.cfg and (node.cfg.kind == p.SHAREDLIB or node.cfg.kind == p.STATICLIB) then
  return "Frameworks"
else
  return categories[path.getextension(node.name)]
end
```

Reply:
As I write in the PR, I kept the extension-based test first to avoid possible breaking changes. However, I agree your suggestion is probably the correct one. If everyone is ok with the early return, I will change the code that way.

Reply:
@starkos thoughts on using the early return instead of the fallback? I'm happy with either for what it's worth.

Reply:
I agree on the early fallback.